embedo.js

```diff
@@ -1315,6 +1322,44 @@
     });
   };
 
+  /**
+   * @method spotify
+   * Spotify oEmbed prototype (Undocumented)
+   *
+   * @param {number} id
+   * @param {HTMLElement} element
+   * @param {string} url
+   * @param {object} options Optional parameters.
+   * @return callback
+   */
+  Embedo.prototype.spotify = function (id, element, url, options, callback) {
+    var size = Embedo.utils.dimensions(element, options.width, options.height);
+    var embed_options = Embedo.utils.merge({
+        url: url
+      },
+      options,
+      Embedo.defaults.RESTRICTED
+    );
+    var embed_uri =
+      Embedo.defaults.SOURCES.spotify.oEmbed + '?' + Embedo.utils.querystring(embed_options);
+
+    Embedo.utils.fetch(embed_uri, function (error, content) {
```

Comment:
I was trying .ajax, however Spotify did not provide proper CORS header upon response and it cause error.